internify/backend/models/UserData.js

```diff
@@ -0,0 +1,53 @@
+var JobPostingData = require("./JobPostingData");
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+    name: {
+        type: String,
+    },
+    email: {
+        type: String,
+    },
+    password: {
+        type: String,
+    },
```

Comment:
I don't think we need to store password in our db. It's being handled by Firebase. I don't think it would be safe to store passwords on our end anyway. Thoughts?

Reply:
agreed, please remove @ReneReid